packages/pwa-kit-react-sdk/src/ssr/server/express.js

```diff
@@ -332,6 +341,62 @@ export const createApp = (options) => {
     return app
 }
 
+/**
+ * Returns the express app configuration file in object form. The object resolution will
+ * be as follows (from highest to lowest priority):
+ *
+ * > {target_name}.ext
+ * > local.ext
+ * > default.ext
+ * > package.json (mobify.key)
+ *
+ * Each file marked with `ext` can optionally be terminated with `yml`, `yaml` or
+ * `json` in that priority.
+ *
+ * NOTE: This is an isomorphic function, when run on the browser the config returned will
+ * be the value serialized in the html.
+ *
+ * @returns - the application configuration object.
+ */
+/* istanbul ignore next */
+const loadConfig = () => {
+    const isRemote = Object.prototype.hasOwnProperty.call(process.env, 'AWS_LAMBDA_FUNCTION_NAME')
+    let moduleName = process?.env?.DEPLOY_TARGET || ''
+
+    // Search options.
+    const searchPlaces = [
+        `config/${moduleName}.yml`,
+        `config/${moduleName}.yaml`,
+        `config/${moduleName}.json`,
+        `config/local.yml`,
+        `config/local.yaml`,
+        `config/local.json`,
+        `config/default.yml`,
+        `config/default.yaml`,
+        `config/default.json`,
+        `package.json`
```

Comment:
Before we merge this, I think we should document the relationship between these different config files. At a minimum, it's not clear to me how `local` and `default` relate to one-another...

Reply:
I've left some more details on these files and when they are used.